packages/ChangesReloaded-Core.package/CRChangeListItem.class/instance/doesNotUnderstand..st

```diff
@@ -0,0 +1,4 @@
+accessing
+doesNotUnderstand: aMessage
+	
+	^ self changeRecord perform: aMessage selector
```

Comment:
Maybe add a comment? A comment is useful in this case because you don't describe what you do (the code is quite clear in that regard) but rather _why_ you do it.
```suggestion
	"Support all methods of the changeRecord by delegating messages that we don't understand to it."
	^ self changeRecord perform: aMessage selector
```